.github/ISSUE_TEMPLATE/release_new_version.md

```diff
@@ -0,0 +1,70 @@
+---
+name: 'For maintainers: Release a new version'
+about: This is the Developer-only template which we use for tracking the process of future releases.
+title: 'Prepare Release 3.y.z'
+labels: release
+assignees: ''
+---
+
+**Target timeline**
+Schuled feature freeze / Start of translation process: <!-- Add a date, usually about 6-8 weeks from the last release, e.g. 2021-04-20 -->
+Approximate release date: <!-- Usually 2-3 weeks after starting translations, e.g. 2021-05-04 -->
+
+**Checklist**
+- [ ] Assign this issue to the release shepherd who is in charge of managing this checklist
+- [ ] Create a milestone and add all relevant issues
+- [ ] Ensure that all issues/PR targeted for this release are done by checking the milestone
+- [ ] Declare a code freeze. PRs can still be worked on and may get reviewed, but must not be merged unless agreed explicitly.
+- [ ] Write the Changelog
+- [ ] [Tag a beta release](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Announce the beta release on Github Discussions
```

Comment:
Hrm, looking at the previous [rc announcement](https://github.com/jamulussoftware/jamulus/discussions/1217), it looks like most (all?) of the replies for relevant to the release candidate. I'd rather tend to not lock these as it makes it easier to get feedback. I think something like "I have made a public testserver with 3.7.0rc1 in Düsseldorf, Genre Rock, named Testserver 3.7.0rc1." is easily posted to such a topic (and provides relevant information), but I suspect the author would not have gone the way to open a new discussion for that?

Reply:
See my comment below, but it's just better to have people post separate topics rather than use the release announcement as a sort of forum within a forum so that we can spot and manage things easier.

Reply:
Yeah, I agree for the final release announcement which received too many unrelated replies. I'm mainly wondering if we should handle pre-releases differently and keep them unlocked.

Reply:
Yes, interim releases before the final release announcement should have comments on in that case.